backend/fms_core/services/library.py

```diff
@@ -0,0 +1,103 @@
+from django.core.exceptions import ValidationError
+
+from fms_core.models import LibraryType, Library
+
+from fms_core.services.process import create_process
+from fms_core.services.container import create_container, get_container
+from fms_core.services.sample import transfer_sample
+from fms_core.services.property_value import create_process_properties
+
+from datetime import datetime
+
+
+def get_library_type(name):
+    instrument = None
+    errors = []
+    warnings = []
+    try:
+        library_type = LibraryType.objects.get(name=name)
+    except LibraryType.DoesNotExist as e:
+        errors.append(f"No Experiment Type named {name} could be found.")
+    except LibraryType.MultipleObjectsReturned as e:
+        errors.append(f"More than one Experiment Type was found with the name {name}.")
+
+    return instrument, errors, warnings
+
+
+def create_library_batch(library_type, library_size, platform, library_date,
```

Comment:
I would call this prepare_library on the same model as extract_sample and transfer_sample...